src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+
+    @Advanced
+    @Argument(doc = "indicate evaluation is being performed on non-human calls",
+            shortName = "nonhs",
+            fullName = "non-human",
+            optional = true)
+    private Boolean nonHumanVCF = false;
+
+    @Advanced
+    @Argument(doc = "string value for chromosome Y; takes value of \"Y\" or \"chrY\", or when left unprovided, " +
+            "evaluation is skipped on chromosome Y; applicable only when \"non-human\" is false",
+            shortName = "chry",
+            fullName = "chr-y-string",
+            optional = true)
+    private String chrYString;
+
+    /**
+     * A fixed amount of padding for each VCF record from the eval call set.
+     * Each individual record might be padded (more or less bases) as judged appropriate by the tool itself.
+     */
+    @Argument(doc = "Padding around variants for overlapping",
+            fullName = "padding",
+            optional = true)
+    private Integer fixedPadding = 50;
+
+    @Argument(doc = "prefix for output",
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
+    private String outPrefix;
+
+    @Argument(doc = "Minimum reciprocal overlap fraction for two concordant variants of the type DEL and INV",
+            fullName = "recip-ovp-frac",
+            optional = true)
+    private Float recipOvpFrac = 0.5f;
+
+    @Argument(doc = "Maximum difference in length between two concordant variants or type INS and/or DUP, " +
+            "i.e. the difference in their length cannot exceed this fraction of the shorter of the two",
+            fullName = "max-ins-len-diff-frac",
+            optional = true)
+    private Float insLenDiffFrac = 0.1f;
+
+    @Advanced
+    @Argument(doc = "Minimum mapping quality of the evidence contig mappings for a eval VCF record to be considered;" +
+            " setting to zero turns off this filter",
+            fullName = "min-map-qual",
+            optional = true)
+    private Integer minMQ = 0;
+
+    @Advanced
+    @Argument(doc = "Minimum alignment length of the evidence contig mappings; setting to zero turns off this filter",
+            fullName = "min-aln-len",
+            optional = true)
+    private Integer minAlnLen = 0;
+
+    // =================================================================================================================
+    private static final String VARIANT_SKIPPED_EVAL_REASON_KEY = "EVAL_SKIP_REASON";
+
+    private static final Set<String> CHROMOSOME_Y_NAMES = Sets.newHashSet("Y", "chrY");
+    private Set<String> chromosomesOfInterest = new HashSet<>();
+
+    // TODO: 6/4/18 this is initialized in a way that $ 1 - insLenDiffScoreCoeff * lenDiff/Math.min(evalLen, truthLen) < recipOvpFrac $ for discordant insertion/duplication
+    private Float insLenDiffScoreCoeff;
+
+    private final List<EvalVariant> classifiedVariants = new ArrayList<>(50_000); // guess
```

Comment:
Why initialize them here and not latter when you may be able to make a better guess?

Reply:
It's file based, so unless I can some how estimate the line count of the call set from the VCF, this is the best option?